### ISSUE TITLE
"Related content" widget is empty on pages

### ISSUE THREAD

**Issue by liddiard** [dailybruin/caeruleum#102]
Any pages that use a template with the sidebar (like http://dailybruin.com/about/ and http://dailybruin.com/the-daily-bruin-editorial-board/) show the related content widget, where it says "Read more for" followed by nothing ([screenshot](https://puu.sh/v3I8e/dc00ef59e2.png)). 

The widget should probably show up on posts only, not on pages.

**Comment by H-Huang** [comment]
is there anyway to modify the type of sidebar that we use for those pages, because for the category pages on the site we don't run into that problem for some reason http://dailybruin.com/category/news/.

**Comment by nathanmsmith** [comment]
I think you can modify which widgets are displayed for a given page in the Wordpress admin panel under [Appearance->Widgets](http://dailybruin.com/wp-admin/widgets.php).

<img width="160" alt="screen shot 2017-03-30 at 6 08 22 pm" src="https://cloud.githubusercontent.com/assets/5677971/24532312/65afcd5a-1574-11e7-9cf4-5fc14924da7e.png">

I'm getting a `Google_AuthException thrown` error whenever I try to access this page though, so that's a bit problematic.

<img width="494" alt="screen shot 2017-03-30 at 6 08 41 pm" src="https://cloud.githubusercontent.com/assets/5677971/24532316/6d92545c-1574-11e7-8b30-5524948f6ff4.png">

[This documentation](https://codex.wordpress.org/WordPress_Widgets) might help also?

**Comment by H-Huang** [comment]
yeah I need to ask upper to reset our analytics credentials they must be expired or something. If you want to get past this and view the widgets, you can temporarily disable our google analytics plugin (just remember to enable it back after finished).
